oci/main.py

```diff
@@ -56,6 +56,7 @@ def system_tests(args):
     cfg = config.load()
 
     suite_type = "basic"
+    coverage = "yes"
```

Comment:
This should be command line option like --coverage. I'm not sure what if this should be enabled by default, since coverage may affect the code under test. It is safer to not enable coverage, and enable it only when we want.